component/GroupCell.tsx

```diff
@@ -106,6 +106,7 @@ export default function Dataset(props: {
     category: string;
     create_time: string;
     department: string;
+    user_name: string;
```

Comment:
adding username to dataset cell display for better user experience
```suggestion
    user_name: string;
```